### PR TITLE
fixes a small bug in bottler

### PR DESCRIPTION
**Pull request by Tlaltecuhtli**
:cl:
fix: bottler can accept reagent chamber's output
sprite: the output pixel is blue instead of red so its in line with the rest of the sprites
/:cl:
weird bug wanted a ..()  might test more edge cases later

### CODE REVIEW

File: code/modules/plumbing/plumbers/bottler.dm [hunk 1]
```diff
@@ -67,13 +67,15 @@
...
 		if(istype(AM, /obj/item/slime_extract)) ///slime extracts need inject
 			AM.forceMove(goodspot)
 			reagents.trans_to(AM, reagents.total_volume, transfered_by = src, method = INJECT)
-			return
+			return TRUE
 		if(istype(AM, /obj/item/slimecross/industrial)) ///no need to move slimecross industrial things
 			reagents.trans_to(AM, reagents.total_volume, transfered_by = src, method = INJECT)
-			return
+			return TRUE
+	..()
```
Comment (AnturK) [inline code/modules/plumbing/plumbers/bottler.dm:81]: There's no parent that processes here.

File: code/modules/plumbing/plumbers/bottler.dm [hunk 2]
```diff
@@ -49,15 +49,15 @@
 ///changing input ammount with a window
 /obj/machinery/plumbing/bottler/interact(mob/user)
 	. = ..()
-	var/vol = min(100, round(input(user,"maximum is 100u","set ammount to fill with") as num|null, 1))
+	var/vol = max(1, min(100, round(input(user,"maximum is 100u","set ammount to fill with") as num|null, 1)))
```
Comment (AnturK) [inline code/modules/plumbing/plumbers/bottler.dm:52]: Could switch this to clamp

**Review by AnturK — CHANGES_REQUESTED** [review]
